.editorconfig

```diff
@@ -0,0 +1,26 @@
+root = true
+
+[*]
+charset = utf-8
+
+[*.cs]
+# spacing rules
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+# formatting rules
+dotnet_style_qualification_for_field = false:error
+dotnet_style_qualification_for_property = false:error
+dotnet_style_qualification_for_method = false:error
+dotnet_style_qualification_for_event = false:error
+dotnet_style_predefined_type_for_locals_parameters_members = true:error
+dotnet_style_predefined_type_for_member_access = true:error
+dotnet_style_require_accessibility_modifiers = always:error
+csharp_style_var_when_type_is_apparent = true:error
+csharp_prefer_braces = true:error
+csharp_using_directive_placement = outside_namespace:error
```

Comment:
setting verbosity levels to "error" gives us `--fix-style error` ability, which basically means `dotnet-format` would fix them for us automatically as it did here in this PR.